runtime.go

```diff
@@ -119,7 +119,7 @@ func (f *stackFrame) write(b *bytes.Buffer) {
 	if f.prg != nil {
 		if n := f.prg.funcName; n != "" {
 			b.WriteString(n)
-			b.WriteString(" (")
+			b.WriteString(" ")
```

Comment:
This is an unrelated change and should not really be part of this PR.

Reply:
I don't really think this is unrelated, as real lines/column number are a based requirement for meaningful debugging in THE golang ide right now (which obviously is Goland ;-)), anyhow gonna remove it.